<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view_todo_list</class>
 <widget class="QMainWindow" name="view_todo_list">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>todo list</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/hr logo.png</normaloff>icons/hr logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(21, 29, 47);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#frame{
background-color: rgb(31, 41, 64);
 border-radius: 5px;
color:rgb(255,255,255);
	
}
QLabel,QCheckBox{

color: rgb(220, 220, 220);
}


#edit{
border: 2px solid rgb(198, 116, 69);
color: rgb(220, 220, 220);
}
#delete_item{
border: 2px solid rgb(218, 0, 3);
color: rgb(220, 220, 220);
}
#add_new_item{
border: 2px solid rgb(45, 208, 0);
color: rgb(220, 220, 220);
}
#edit:pressed , #delete_item:pressed , #add_new_item:pressed{
color: rgb(255, 255, 255);
	background-color: rgb(21, 29, 47);
}










QLabel,QCheckBox{
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.243781 rgba(62, 62, 62, 0));
}


QLineEdit , QPushButton , QToolButton , QListWidget , QListWidget::item , QPushButton , QDateEdit,QTimeEdit
,QTextEdit
{
padding-left:6px;
padding-right:6px;
color:rgb(255,255,255);
border: 1px solid rgb(48, 64, 99);;
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.243781 rgba(62, 62, 62, 0));
border-radius: 4px;

}


QListWidget{
background-color: rgb(23, 31, 49);

}


QPushButton , QToolButton,QLineEdit,QDateEdit,QTimeEdit{

border-radius: 10px;}




QScrollBar::sub-page:vertical ,QScrollBar::add-page:vertical{
background:rgb(24, 32, 49);}


QToolButton:pressed,QPushButton:pressed,QListWidget:item {
padding:1px;
background-color: rgb(20, 27, 45);}


QListWidget {
padding-top:8px;

}
 QPushButton , QToolButton{

padding:0px;
}


QScrollBar::handle:vertical,QListWidget:item:hover{
border-radius: 4px;
border: 2px solid rgb(164, 190, 217);
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.243781 rgba(62, 62, 62, 0));
}


QListWidget::item:selected {
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.243781 rgba(62, 62, 62, 0));

border: 2px solid rgb(17, 0, 255);
color:rgb(10, 57, 170);
}






QToolTip{
color:rgb(210,210,210);
background-color:rgb(42, 58, 94);
border-radius: 7px;
border: 2px solid rgb(0, 112, 187);
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="2,2,1,2,0,2,0,2,2">
       <item row="1" column="3" colspan="4">
        <widget class="QDateEdit" name="date">
         <property name="minimumSize">
          <size>
           <width>74</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Yu Gothic UI Semibold</family>
          </font>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="5" colspan="2">
        <widget class="QListWidget" name="todo_items">
         <property name="font">
          <font>
           <family>Yu Gothic UI Semibold</family>
          </font>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>New Item</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Yu Gothic UI Semibold</family>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Title :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Yu Gothic UI Semibold</family>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;to :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QTimeEdit" name="time_to">
         <property name="minimumSize">
          <size>
           <width>74</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Yu Gothic UI Semibold</family>
          </font>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="6">
        <widget class="QLabel" name="total_time">
         <property name="font">
          <font>
           <family>Yu Gothic UI Semibold</family>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="3" colspan="4">
        <widget class="QLineEdit" name="title">
         <property name="minimumSize">
          <size>
           <width>74</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Yu Gothic UI Semibold</family>
          </font>
         </property>
         <property name="placeholderText">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="7">
        <widget class="QTextEdit" name="discription">
         <property name="font">
          <font>
           <family>Yu Gothic UI Semibold</family>
          </font>
         </property>
         <property name="placeholderText">
          <string>Discription</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Yu Gothic UI Semibold</family>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Date :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QTimeEdit" name="time_from">
         <property name="minimumSize">
          <size>
           <width>74</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Yu Gothic UI Semibold</family>
          </font>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Yu Gothic UI Semibold</family>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;from :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QCheckBox" name="done">
         <property name="font">
          <font>
           <family>Yu Gothic UI Semibold</family>
          </font>
         </property>
         <property name="text">
          <string>Done</string>
         </property>
        </widget>
       </item>
       <item row="1" column="7">
        <widget class="QCheckBox" name="everyday">
         <property name="font">
          <font>
           <family>Yu Gothic UI Semibold</family>
          </font>
         </property>
         <property name="text">
          <string>Every Day</string>
         </property>
        </widget>
       </item>
       <item row="2" column="7">
        <widget class="QCheckBox" name="auto_del">
         <property name="text">
          <string>Auto Delete</string>
         </property>
        </widget>
       </item>
       <item row="2" column="8">
        <widget class="QPushButton" name="edit">
         <property name="minimumSize">
          <size>
           <width>74</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
       </item>
       <item row="1" column="8">
        <widget class="QPushButton" name="delete_item">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>74</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Yu Gothic UI Semibold</family>
          </font>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QPushButton" name="add_new_item">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>74</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Yu Gothic UI Semibold</family>
          </font>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>title</tabstop>
  <tabstop>date</tabstop>
  <tabstop>time_to</tabstop>
  <tabstop>time_from</tabstop>
  <tabstop>auto_del</tabstop>
  <tabstop>everyday</tabstop>
  <tabstop>done</tabstop>
  <tabstop>discription</tabstop>
  <tabstop>add_new_item</tabstop>
  <tabstop>delete_item</tabstop>
  <tabstop>edit</tabstop>
  <tabstop>todo_items</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
